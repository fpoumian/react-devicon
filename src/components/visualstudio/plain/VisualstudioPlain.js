
import React from 'react'
import PropTypes from 'prop-types'
import SVGInline from 'react-svg-inline'
import iconSVG from './VisualstudioPlain.svg'
import { normalizeUnit } from '../../utils.js'

/** VisualstudioPlain */
function VisualstudioPlain({width, height}) {
  return (
     <SVGInline
      classSuffix="-devicon"
      className="VisualstudioPlain"
      svg={iconSVG}
      width={normalizeUnit(width)}
      height={normalizeUnit(height)}
    />
  )
}

VisualstudioPlain.propTypes = {
  width: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
  height: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
}

export default VisualstudioPlain
