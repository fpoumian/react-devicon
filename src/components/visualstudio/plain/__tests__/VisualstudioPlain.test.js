import React from 'react'
import renderer from 'react-test-renderer'
import VisualstudioPlain from '../index.js'

it('renders correctly', () => {
  const tree = renderer.create(<VisualstudioPlain />).toJSON()
  expect(tree).toMatchSnapshot()
})
