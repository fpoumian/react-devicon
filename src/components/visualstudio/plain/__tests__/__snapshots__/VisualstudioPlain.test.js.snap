// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="SVGInline VisualstudioPlain"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<svg class=\\"SVGInline-devicon VisualstudioPlain-devicon\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 128 128\\"><defs><style>.a{fill:#68217a}</style></defs><title>visualstudio-plain</title><path class=\\"a\\" d=\\"M95 2.3l30.5 12.3v98.7l-30.7 12.4-49-48.7-31 24.1-12.3-6.2V33.1l12.3-5.9 31 24.3zM14.8 45.7v37.5l18.5-19zm48.1 18.5l31.9 25.1V39z\\"/></svg>",
    }
  }
/>
`;
